From dfab0b6ea58a15561f471242a096c7cca5ccc461 Mon Sep 17 00:00:00 2001
From: Demetris Ierokipides <ierokipides.dem@gmail.com>
Date: Thu, 22 Dec 2022 05:14:20 +0200
Subject: [PATCH] MiQi: Overclock GPU to 700mhz Since now the gpu regulator is 
 properly set we are allowed to tune the gpu frequencies adding another 100mhz
  without any overheating or other issues

---
 arch/arm/boot/dts/rk3288-miqi.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-miqi.dts b/arch/arm/boot/dts/rk3288-miqi.dts
index 5f408fb0b..0a8197124 100644
--- a/arch/arm/boot/dts/rk3288-miqi.dts
+++ b/arch/arm/boot/dts/rk3288-miqi.dts
@@ -145,6 +145,12 @@ &gpu {
 	status = "okay";
 };
 
+&gpu_opp_table {
+		opp-700000000 {
+			opp-hz = /bits/ 64 <700000000>;
+			opp-microvolt = <1350000>;
+		};
+
 &hdmi {
 	ddc-i2c-bus = <&i2c5>;
 	status = "okay";
-- 
2.34.1

